archinstall/lib/translationhandler.py

```diff
@@ -190,10 +191,7 @@ def __add__(self, other) -> DeferredTranslation:
 	def format(self, *args) -> str:
 		return self.message.format(*args)
 
-	@classmethod
-	def install(cls) -> None:
-		import builtins
-		builtins._ = cls  # type: ignore[attr-defined]
 
+builtins._ = DeferredTranslation  # type: ignore[attr-defined]
```

Comment:
How is this change relevant to the pull request?

Reply:
To be able to use the translation `_` in the args.py file it needs to be initialized which is done in the __init__.py file which is an explicit call AFTER all imports are resolved. 
But because the args.py file is loaded before that happens the `_` has not been registered in the builtins yet and therefore it won't be available for the initial decryption prompt.